changelog/Microsoft.DirectoryServices.json

```
@@ -54,15 +54,59 @@
     },
     {
       "ChangeList": [
-      { 
-        "Id": "e114cd2d-6d20-498d-a61e-623baee7e0f1",
-        "ApiChange": "Method",
-        "ChangedApiName": "user",
-        "ChangeType": "Addition",
-        "Description": "Added the [changePassword](https://docs.microsoft.com/en-us/graph/api/user-changepassword?view=graph-rest-1.0) action to the [user](https://docs.microsoft.com/en-us/graph/api/resources/user?view=graph-rest-1.0) resource.",
-        "Target": "user"
-      }
-    ],
+        {
+          "Id": "2dca7e8b-c8af-45c1-9fa6-f108e93d8562",
+          "ApiChange": "Resource",
+          "ChangedApiName": "policies",
+          "ChangeType": "Addition",
+          "Description": "Added navigation properties 'defaultAppManagementPolicy' and 'appManagementPolicies' to policies singleton.",
```

- Add the public link to policies resources.
- Is it to policies or to policyRoot? Kindly confirm

```suggestion
          "Description": "Added the **defaultAppManagementPolicy** and **appManagementPolicies** navigation properties to the [policies](add public link) singleton.",
```

---

updated.